docs/assets/js/charts.js

```diff
@@ -554,10 +554,14 @@ function createTable(table)
                     return d;
                 });
 
+            let displayed = data;
+            if (hasConfig($(table), 'slice'))
+                displayed = data.slice(readConfig($(table), 'slice')[0], readConfig($(table), 'slice')[1]);
```

Comment:
minor nits:
Maybe move this into a variable?
```
let tableSlice = readConfig($(table), 'slice');
``
Maybe rename `displayed` to `displayData`. `displayed` confused me because of the past tense.

@pluehne what do you think?

Reply:
I agree with @larsxschneider, `displayed` could be renamed to `displayData` for clarity what is actually displayed.

I believe that I have similar code with two `readConfig` calls as parameters on the same line somewhere else in the code ([here](https://github.com/mlbright/hubble/blob/ab2bf6c05d37289d69be2880d2d5e73fc9867b6d/docs/assets/js/charts.js#L454) and [here](https://github.com/mlbright/hubble/blob/ab2bf6c05d37289d69be2880d2d5e73fc9867b6d/docs/assets/js/charts.js#L559)), so I’ll take the blame for that 🙂. However, this might be the occasion to make the change at the other places as well.

Reply:
In this case, we should make the `tableSlice` constant. Also, I’d rather call it `configSlice` or something similar to show that it’s part of the configuration and not a slice of the table containing actual data.

Reply:
I made the grammar change, but I'll leave the constant tableSlice changes for a separate PR.